examples/slackcontroller/README.md

```diff
@@ -0,0 +1,14 @@
+# SlackController
+
+A sample controller that demonstrates how to write a controller that targets the
+cluster registry. This controller will post messages to a Slack channel when a
+cluster is added to or removed from the registry.
+
+## Quickstart
+
+1.  Set up a cluster registry. You can use [crinit](cmd/crinit/crinit.go) to do
+    so.
+1.  Run `bazel build //examples/slackcontroller` to build the cluster registry.
```

Comment:
This build the controller, not the registry?

Reply:
D'oh! Of course, fixed.